projeto_guiado_v2_williane.py

```diff
@@ -0,0 +1,187 @@
+# Definindo o dataset diretamente no programa
+dataset = {}
```

Comment:
O dataset já estava definido e importado, essa parte não precisava ter alterado. Do modo como está, ele não está recuperando os dados do dicionário que estava existente. Quando usamos o import, conseguimos usar os dados e funções de outros arquivos python no nosso arquivo atual.

**Sugestão:**
```suggestion
from dataset_alunas import dataset
```

Neste caso, é só usar o dataset igual usamos em sala de aula;